polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0
+    for exp in range(0, len(expression)-1):
+        exp_list.append(exp)
+
+    reverse_exp_list = exp_list[::-1]
+    temp = 0
+
+    for item in reverse_exp_list:
+        if item == "+":
+            operand1 = s.pop()
+            operand2 = s.pop()
+            result = operand1 + operand2
+            s.push(result)
+        elif item == "-":
+            operand1 = s.pop()
+            operand2 = s.pop()
+            result = operand1 - operand2
+            s.push(result)
+        elif item == "*":
+            operand1 = s.pop()
+            operand2 = s.pop()
+            result = operand1 * operand2
+            s.push(result)
+        elif item == "/":
+            operand1 = s.pop()
+            operand2 = s.pop()
+            result = operand1 / operand2
+            s.push(result)
+        elif item is in [0123456789]:
+            s.push(item)
+        else:
+            print("Invalid input")

```

Comment:
That is a silent failure: https://softwareengineering.stackexchange.com/questions/190528/error-handling-should-a-program-fail-on-errors-or-silently-ignore-them

You don't expect this input and as such should raise an exception. You are discarding the invalid input as if it didn't exist. Look at this example:
input = "+ 3 % 5"
output = 8